go.mod

```diff
@@ -5,49 +5,59 @@ go 1.13
 	github.com/fsnotify/fsnotify v1.4.9 // indirect
 	github.com/go-kit/kit v0.10.0
 	github.com/go-sql-driver/mysql v1.5.0
 	github.com/gorilla/mux v1.8.0
 	github.com/hashicorp/go-multierror v1.1.0 // indirect
 	github.com/hashicorp/go-rootcerts v1.0.2 // indirect
-	github.com/jaegertracing/jaeger-lib v2.2.0+incompatible
-	github.com/jlaffaye/ftp v0.0.0-20200715164256-5d10dd64f695
-	github.com/klauspost/compress v1.10.10 // indirect
+	github.com/jaegertracing/jaeger-lib v2.4.0+incompatible
+	github.com/jlaffaye/ftp v0.0.0-20200812143550-39e3779af0db
+	github.com/klauspost/compress v1.11.1 // indirect
 	github.com/lopezator/migrator v0.3.0
+	github.com/magiconair/properties v1.8.4 // indirect
 	github.com/mattn/go-sqlite3 v2.0.3+incompatible
 	github.com/moov-io/ach v1.4.4
-	github.com/moov-io/base v0.11.1
-	github.com/moov-io/customers v0.5.0-dev2
+	github.com/moov-io/base v0.11.2-0.20201009172605-8a2f4aef9cc4
+	github.com/moov-io/customers v0.5.0-rc4.0.20201013202949-f2a316c3b0fe
+	github.com/moov-io/identity v0.2.8 // indirect
 	github.com/opencontainers/go-digest v1.0.0 // indirect
 	github.com/opentracing/opentracing-go v1.2.0
-	github.com/ory/dockertest/v3 v3.6.0
+	github.com/ory/dockertest/v3 v3.6.1
 	github.com/ory/mail/v3 v3.0.0
-	github.com/pelletier/go-toml v1.8.0 // indirect
-	github.com/pkg/sftp v1.11.0
+	github.com/pelletier/go-toml v1.8.1 // indirect
+	github.com/pkg/sftp v1.12.0
 	github.com/prometheus/client_golang v1.7.1
 	github.com/robfig/cron/v3 v3.0.1
-	github.com/sirupsen/logrus v1.6.0 // indirect
-	github.com/spf13/afero v1.3.2 // indirect
+	github.com/spf13/afero v1.4.1 // indirect
 	github.com/spf13/cast v1.3.1 // indirect
 	github.com/spf13/jwalterweatherman v1.1.0 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
 	github.com/spf13/viper v1.7.1
 	github.com/uber/jaeger-client-go v2.25.0+incompatible
-	github.com/uber/jaeger-lib v2.2.0+incompatible
+	github.com/uber/jaeger-lib v2.4.0+incompatible
+	go.uber.org/atomic v1.7.0 // indirect
 	gocloud.dev v0.20.0
 	gocloud.dev/pubsub/kafkapubsub v0.20.0
 	gocloud.dev/secrets/hashivault v0.20.0 // indirect
 	goftp.io/server v0.4.0
-	golang.org/x/crypto v0.0.0-20200820211705-5c72a883971a
-	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
+	golang.org/x/crypto v0.0.0-20201012173705-84dcc777aaee
+	golang.org/x/oauth2 v0.0.0-20200902213428-5d25da1a8d43
+	golang.org/x/sync v0.0.0-20201008141435-b3e1573b7520 // indirect
 	golang.org/x/text v0.3.3
 	golang.org/x/time v0.0.0-20200630173020-3af7569d3a1e // indirect
-	gopkg.in/ini.v1 v1.57.0 // indirect
+	golang.org/x/tools v0.0.0-20201013201025-64a9e34f3752 // indirect
+	gopkg.in/ini.v1 v1.62.0 // indirect
 	gopkg.in/square/go-jose.v2 v2.5.1 // indirect
+	honnef.co/go/tools v0.0.1-2020.1.5 // indirect
 )
+
+replace goftp.io/server v0.4.0 => github.com/adamdecaf/goftp-server v0.4.0
+
+exclude github.com/moov-io/identity v0.2.8
+exclude github.com/moov-io/identity v0.2.9
```

Comment:
But of course, now that you've excluded these versions it's complaining about 0.3.1 🙄 
> ... reading github.com/moov-io/identity/go.mod at revision v0.3.1: unknown revision v0.3.1

Reply:
I think we need to get identity out of paygate's dependency tree in order to remove these `exclude` statements.

Reply:
Sure seems that way. We should probably make sure we don't have private dependencies in other public repos too...

Reply:
Actually. I think it's from the OpenShift image..

Reply:
Nice find! The build finally passed :D